src/debugger/debugFeature.ts

```diff
@@ -42,6 +52,11 @@ export class JuliaDebugFeature {
     public dispose() { }
 }
 
+const getActiveUri = (
+    uri: vscode.Uri | undefined,
+    editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor
+) => uri ? uri.fsPath : editor ? editor.document.fileName : undefined
```

Comment:
Maybe write this as `function getActiveUri(...) {...}` ?

Reply:
sure, if you prefer that :)